gh-deploy.sh

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/env bash
+
+# TODO
+#===================
+# provide a clean way to define/check the "current" version of node (i.e., the one we should execute the publish under/for)
+## ideally we should read it from .nvmrc
+# for local deployment, instead of using stark-ssh.enc, we should use the GITHUB_API_KEY key passed via --github-api-key=foo
+## if present then we could push directly without handling decryption, etc
+
+set -u -e -o pipefail
+
+VERBOSE=false
+TRACE=false
+DRY_RUN=false
+ENFORCE_SHOWCASE_VERSION_CHECK=true
+SOURCE_BRANCH="master"
+TARGET_BRANCH="gh-pages"
+COMMIT_HASH=`git rev-parse --verify HEAD`
+
+TARGET_REPO="git@github.com:NationalBankBelgium/stark.git"
+EXPECTED_REPO_SLUG="NationalBankBelgium/stark"
+EXPECTED_NODE_VERSION="8"
+
+COMMIT_AUTHOR_USERNAME="TravisCI"
+COMMIT_AUTHOR_EMAIL="seb@dsebastien.net"
```

Comment:
Should not we create a specific email address (NBB) for Stark on GitHub ?

Reply:
Yes but no :)